apps/obs-web/src/views/DebrieferAssessmentEditor.vue

```diff
@@ -0,0 +1,348 @@
+                        <!-- <span v-if="props.rows.options.indexOf(option) === 0">
+                          good option
+                        </span> -->
+                        <span v-if="props.row && props.row.options.indexOf(option) === 0" style="font-size: .6em;">'OK/good' option</span>
+                        <q-btn icon="clear" flat @click="props.row.options.splice(props.row.options.indexOf(option), 1)"></q-btn>
+                      </template>
+                    </q-input>
+                  </div>
+                    <q-btn flat style="float: right; margin-top: 10px" @click="props.row.options.push('')">add option</q-btn>
+                </q-popup-edit>
+              </q-td>
+              <q-td key="isWcgop" :props="props">
+                {{ props.row.isWcgop }}
+                <q-popup-edit v-model="props.row.isWcgop" buttons persistent @save="save(props.row)">
+                    <q-select v-model="props.row.isWcgop" :options="[true, false]" dense></q-select>
+                </q-popup-edit>
+              </q-td>
+              <q-td key="isAshop" :props="props">
+                {{ props.row.isAshop }}
+                <q-popup-edit v-model="props.row.isAshop" buttons persistent @save="save(props.row)">
+                    <q-select v-model="props.row.isAshop" :options="[true, false]" dense></q-select>
+                </q-popup-edit>
+              </q-td>
+            </q-tr>
+          </template>
+        </q-table>
+
+    </div>
+</template>
+
+<script lang="ts">
+import {
+  createComponent,
+  ref,
+  onMounted,
+  watch,
+  set,
+  computed
+} from '@vue/composition-api';
+
+import { couchService } from '@boatnet/bn-couch';
+import { Client } from 'davenport';
+import { orderBy, uniq, flatten, sortBy } from 'lodash';
+import moment from 'moment';
+import { authService } from '@boatnet/bn-auth';
+
+export default createComponent({
+  props: {
+    infoTab: Boolean,
+    isFullSize: Boolean,
+  },
+  setup(props, context) {
+    const store = context.root.$store;
+    const state = store.state;
+    const masterDB: Client<any> = couchService.masterDB;
+
+    const debriefer = state.debriefer;
+
+    const jp = require('jsonpath');
+
+    const questionsPagination = {rowsPerPage: 15};
+    const answerSetsPagination = {rowsPerPage: 15};
+
+    const sections: any = ref([]);
+
+    const answerSetNames: any = ref([]);
+
+    const questions: any = ref([]);
+    const questionColumns: any = [
+      { name: 'section', label: 'Section', field: 'section', sortable: true, align: 'left' },
+      { name: 'question', label: 'Question', field: 'question', sortable: true, align: 'left' },
+      { name: 'answerSet', label: 'Answer Set', field: 'answerSet', sortable: true, align: 'left' },
+      { name: 'order', label: 'Order', field: 'order', sortable: true, align: 'center' },
+      { name: 'isWcgop', label: 'WCGOP?', field: 'isWcgop', sortable: true, align: 'center' },
```

Comment:
What is this column for? Does this turn questions on and off for a certain program. If so thinking could show a toggle here instead like they do here. 
![image](https://user-images.githubusercontent.com/44685558/118558635-c9a58500-b71b-11eb-91ae-7263a95c4725.png)

Reply:
yes, isWcgop and isAshop turn questions on and off for the respective program.

It's not a toggle for 2 reasons... 
- the table sorts by program as a default, so if you're in 'ashop' mode, the rows with isAshop appear first - to me this seems more obvious when the column has booleans rather than toggles.
- it should be hard to turn a question on or off for a program - the toggles seem too easy to accidentally flip.

When a new question is created, the boolean for current mode program is automatically set true, so the only time a user would need to change the boolean is when the question is no longer in use, or one program wants to adopt a question from the other.

Suggest we leave as is and get feedback from debriefers.

Reply:
I see okay we can leave it then